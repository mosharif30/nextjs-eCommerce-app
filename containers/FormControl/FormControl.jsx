/* eslint-disable react/prop-types */
import React from 'react'

/** @jsxImportSource @emotion/react */
import { css } from '@emotion/react'
import { useTheme } from '@emotion/react'

function FormControl({ children }) {
  const theme = useTheme()
  return (
    <div
      css={css`
        display: flex;
        padding: ${theme.spaces[3]};
      `}
    >
      {children}
    </div>
  )
}

export default FormControl
