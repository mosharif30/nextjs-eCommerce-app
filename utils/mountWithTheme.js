import { ThemeProvider } from '@emotion/react'
import theme from '../configs/theme'
const mountWithTheme 
