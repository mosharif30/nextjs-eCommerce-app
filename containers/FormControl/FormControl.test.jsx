import { render } from '@testing-library/react'
import mountWithTheme from '../../utils/mountWithTheme'
import FormControl from './FormControl'
describe('FormControl', () => {
  test('render correctly', () => {
    const component = mountWithTheme(<FormControl></FormControl>)
    const tree = component
    expect(tree).toMatchSnapshot()
  })
})
