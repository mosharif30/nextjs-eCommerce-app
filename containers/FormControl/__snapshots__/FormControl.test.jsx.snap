// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormControl render correctly 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-1241p75-FormControl{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;padding:8px;}",
      }
    }
    data-emotion="css 1241p75-FormControl"
  />,
  <div
    className="css-1241p75-FormControl"
  />,
]
`;
